js/app.js

```diff
@@ -4,6 +4,15 @@ import Client from "./client"
 import GameState from "./gamestate"
 import GameView from "./gameview"
 
+
+// toggle verbose logging
+let LOG_VERBOSE = true;
+console.log_verbose = console.log;
```

Comment:
actually - we are going to want game state to be runnable from the server, and this is introducing a global dependency that app.js can be the only entry point to gamestate. can we just not have a log-verbose function for now? is there really too much logging?